tools/distrib/python/submit.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
```

Comment:
It's just a boat float - if it is by consensus preferred to make things shell scripts, I'm perfectly willing to just change it.